app/views/sparql/index.html.erb

```diff
@@ -0,0 +1,90 @@
+<h1>SPARQL Interface</h1>
+
+<div style="background: #f0f0f0; padding: 10px; margin-bottom: 20px;">
+  <h3>Configuration Status:</h3>
+  <% if flash[:warning] %>
+    <p style="color: orange;">⚠️ <%= flash[:warning] %></p>
+  <% else %>
+    <p style="color: green;">✅ SPARQL endpoint is configured and ready</p>
+    <p><small>Endpoint: <%= controller.send(:get_virtuoso_uri) rescue 'N/A' %></small></p>
+    <%= form_with url: query_sparql_index_path, method: :post, local: true, format: :html do |form| %>
+      <div>
+        <label>SPARQL Query:</label><br>
+        <%= form.text_area :sparql_query, value: @sparql_query,
+                           rows: 8,
+                           style: "width: 100%; font-family: monospace;",
+                           placeholder: "Enter your SPARQL query here...\n\nExample:\nSELECT ?subject ?predicate ?object\nWHERE {\n  ?subject ?predicate ?object .\n}\nLIMIT 10" %>
+      </div>
+      <br>
+      <div>
+        <label>Output Format:</label>
+        <%= form.select :format, [['Table', 'table'], ['JSON', 'json'], ['XML', 'xml']], {}, {style: "margin-left: 10px;"} %>
+      </div>
+      <br>
+      <div>
+        <%= form.submit "Execute Query", style: "background: #007bff; color: white; padding: 8px 16px; border: none;" %>
+        <button type="button" onclick="document.querySelector('textarea').value = ''" style="margin-left: 10px; padding: 8px 16px;">Clear</button>
+        
+        <% if @example_queries && @example_queries.any? %>
```

Comment:
tip: you can just call `@example_queries&.any?` without checking if it's present. & is a shortcut for `.try()` which always returns false if the item is nil